src/lib/types.ts

```diff
@@ -38,3 +38,7 @@ export type StorageOwnData = {
     version: number
   }
 }
+
+export type FetchProfileOptions = {
+  useCache: boolean
```

Comment:
As it is a set of options, should we mark them as optional as well?
```suggestion
  useCache?: boolean
```

Reply:
updated!